voila/treehandler.py

```diff
@@ -76,7 +87,8 @@ def allowed_content(content):
                        contents=contents,
                        terminals_available=False,
                        server_root=get_server_root_dir(self.settings),
-                       theme=self.voila_configuration.theme,
+                       theme=theme_arg,
+                       query=self.request.query,
```

Comment:
I'm not sure that we want to pass all queries of the tree page to the notebook URLs, maybe just the theme and template?

Reply:
As of this PR, that's the only two query arguments we support, or do we support more arguments?

Reply:
`self.request.query` contains all the queries passed to the tree handler, and then it is appended to the notebook URLs. I'm thinking about a more limited scope where only  `voila-theme` and `voila-template` are transferred to the URL

Reply:
I think your original comment will make sense if we add some query arguments specific to the TreeHandler at some point. But we can probably filter the query arguments here only when that happens?

Reply:
If `Voila` is served under proxy, there might be some sort of token query for the tree handler, but I think there is no harm in transferring these tokens to the notebook URLs. We can filter the query when we actually use them in the tree handler.